Event.php

```diff
@@ -281,4 +325,40 @@ static public function fromException(\Throwable $e)
         return $ev;
     }
 
+    /**
+     * @param array $error
+     *
+     * @return Event
+     */
+    public static function fromError($error)
+    {
+
+        $ev = new Event();
+        $ev->setError($error);
+        return $ev;
+    }
+
+    protected function setError($error)
+    {
+        $this->data['exception'] = [
+            'values' => [
+                [
+                    'type' => $this->errorTypeToString($error['type']),
+                    'value' => $error['message'],
+                    'stacktrace' => [
```

Comment:
Would there be a way to get an actual stacktrace? Could we call debug_backtrace() and remove all calls that happen somewhere in the sentry plugin?

Reply:
No, unfortunately not. The backtrace returned by `debug_backtrace()` is always only the current function.